src/Computer_Item.php

```diff
@@ -190,6 +190,8 @@ public function cleanDBonPurge()
             $computer = new Computer();
             $computer->getFromDB($this->fields['computers_id']);
 
+            $is_mainitem_dynamic = (bool) ($computer->fields['is_dynamic'] ?? false);
```

Comment:
Shoudl we really have a main item not dynamic during inventory?

Reply:
I suppose the following items can be added manually

```
$CFG_GLPI["directconnect_types"]          = ['Monitor', 'Peripheral', 'Phone', 'Printer'];
```

Reply:
It's not a blocker anyways